ovirt-engine-keycloak.spec.in

```diff
@@ -0,0 +1,113 @@
+%global __jar_repack 0
+
+%global product_name Keycloak SSO for oVirt Engine
+
+%global make_common_opts \\\
+	-j1 \\\
+	BUILD_VALIDATION=0 \\\
+	PACKAGE_NAME=%{name} \\\
+	RPM_VERSION=%{version} \\\
+	RPM_RELEASE=%{release} \\\
+	PREFIX=%{_prefix} \\\
+	SYSCONF_DIR=%{_sysconfdir} \\\
+	DATAROOT_DIR=%{_datadir} \\\
+	%{?EXTRA_BUILD_FLAGS:EXTRA_BUILD_FLAGS="%{EXTRA_BUILD_FLAGS}"}
+
+
+
+########################################################
+#  Keycloak overlay package
+########################################################
+Name:		ovirt-engine-keycloak
+Version:	@RPM_VERSION@
+Release:	@RPM_RELEASE@%{?dist}
```

Comment:
No, that's similar to ovirt-engine-wildfly, we don't do nightly builds, because we usually perform a release only if we bump to a new wildfly release. So every change means new stable release